Compile error:

` [04:22:51 INF] [CONTAINER] /work/repo/Plugin.cs(123,30): error CS1061: 'ChatTimestampService' does not contain a definition for 'ApplyConfiguration' and no accessible extension method 'ApplyConfiguration' accepting a first argument of type 'ChatTimestampService' could be found (are you missing a using directive or an assembly reference?) [/work/repo/Clock.csproj]`

Please make sure your code compiles locally.

--- a/stable/Clock/manifest.toml
+++ b/stable/Clock/manifest.toml
@@ -1,6 +1,6 @@
 [plugin]
 repository = "https://github.com/seventity7/Clock.git"
-commit = "4969541ea139f6b453171bfdae9aa2169c206693"
+commit = "aeb223c7ec5bb2a27c3cc56517346585ebb0164e"
 owners = ["seventity7"]
 maintainers = ["seventity7"]
 project_path = "."

